### ISSUE TITLE
Unable to load remote content in Opera

### ISSUE THREAD

**Issue by nathan-osman** [Benjol/SE-AutoReviewComments#110]
I've installed the script in Opera using the Tampermonkey extension. However, when attempting to fetch a remote source from a GitHub repo., I get the following error in the console:

    Refused to execute script from 'https://raw.github.com...' because its MIME type
    ('text/plain') is not executable, and strict MIME type checking is enabled.

This appears to be a problem with the way GitHub is serving the file - despite an extension of `.js` or `.jsonp`, the content-type is still set to `text/plain`. Is there any way to work around this?

**Comment by Benjol** [comment]
I have to admit I've no idea, perhaps @oliversalzburg has tried importing from a remote source?

**Comment by oliversalzburg** [comment]
Nope, this has never really worked for me and, apparently, is a PITA to implement right. IIRC Mozilla also noted the foreign resource loading as an issue during initial review.

**Comment by hg8** [comment]
No workaround then ?
